core/commands/destroy_command.rb

```diff
@@ -113,18 +114,21 @@ def destroy_by_node_name
     vagrant_cleaner = VagrantCleaner.new(@env, @ui)
     vagrant_vm_list = vagrant_cleaner.vm_list
     aws_vm_list = @aws_service.instances_names_list
+    gcp_vm_list = @gcp_service.instances_list
```

Comment:
From the code in Session we can see that AWS and GCP are optional. This code will fail if user did not provide AWS or GCP credentials.